content/client-lib-development-guide/features.textile

```diff
@@ -1282,8 +1282,10 @@ class RealtimeChannel:
   push: PushChannel
 
 class PushChannel:
-  subscribe(type: .Device | .ClientId) => io
-  unsubscribe(type: .Device | .ClientId) => io
+  subscribeDevice() => io
+  subscribeClientId() => io
```

Comment:
I assume in the spec this should fail if the user is not authenticated with a `clientId`?